spec/controllers/guide/nodes_controller_spec.rb

```diff
@@ -48,7 +48,11 @@
             end
 
             it "does not render the show template" do
-              expect(response).not_to render_template(:show)
+              if Rails.version < '5'
```

Comment:
One idea is we make this version (0.4.0) *only* compatible with Rails 5, and projects on Rails 4 can continue to use 0.3.2. What do you think?

Reply:
@shevaun I like that idea. The future is now!

Reply:
that's a good point. Yep I'm happy to make this version 4+ compatible!